apps/designer/tsconfig.json

```diff
@@ -1,10 +1,35 @@
 {
-  "extends": "tsconfig/nextjs.json",
-  "include": ["next.d.ts", "**/*.ts", "**/*.tsx"],
   "compilerOptions": {
     "types":[
       "babylonjs",
     ],
+    "target": "es5",
```

Comment:
Same issue as the comments above. These changes are incorrect for turborepo. Refer to comment above for how to fix this